lms/templates/instructor/instructor_dashboard_2/e-commerce.html

```diff
@@ -77,6 +81,7 @@ <h3 class="error-msgs" id="error-msg"></h3>
          </div>
       </div><!-- end wrap -->
       %endif
+      %if section_data['is_white_label']:
```

Comment:
Can we change this conditional to be more like 'coupons_supported'? Ideally, coupons aren't really white label features (they work in Open edX outside of white label/microsites).